pkg/flag/flag.go

```diff
@@ -28,23 +28,34 @@ const (
 	TriggerControllerName = "tekton-triggers-controller"
 	TriggerWebhookName    = "tekton-triggers-webhook"
 
-	AnnotationPreserveNS = "operator.tekton.dev/preserve-namespace"
+	AnnotationPreserveNS  = "operator.tekton.dev/preserve-namespace"
+	LabelProviderType     = "operator.tekton.dev/provider-type"
+	ProviderTypeCommunity = "community"
+	ProviderTypeRedHat    = "redhat"
+	ProviderTypeCertified = "certified"
 
 	uuidPath = "deploy/uuid"
 )
 
 var (
 	flagSet *pflag.FlagSet
 
-	TektonVersion   = "v0.9.2"
-	PipelineSA      string
-	IgnorePattern   string
-	ResourceWatched string
-	ResourceDir     string
-	TargetNamespace string
-	NoAutoInstall   bool
-	Recursive       bool
-	OperatorUUID    string
```

Comment:
Shouldn't this be v0.10.x ?

Reply:
it is `v0.10.x` on release-branch,
this feature is being merged to master.

In fact, it should be a placeholder like `devel`. At present the version in master branch is `0.9.2`

Reply:
While I was typing this message, the 0.10.x release branch got merged.
Now you are right, it should be 0.10.1
Rebasing.